### ISSUE TITLE
Unremovable folder on desktop

### ISSUE THREAD

**Issue by benkirton** [keybase/client#7070]
Keybase GUI Version: 1.0.22-20170515141716+b608f0e
No thank you. Please provide a way to place it somewhere else.

**Comment by strib** [comment]
I believe this is usually because you opened the dmg itself on the desktop, instead of dragging it to the Applications folder.

**Comment by benkirton** [comment]
I opened the dmg from my downloads folder when installing. It's now in my Applications folder.
When the app is open and in my toolbar it generates the folder/drive on my desktop and can't be removed until the app is closed.
It may be related to it being a mounted drive and they are shown based on my desktop view settings.
Seems the wrong way to present a folder though.

**Comment by waylonis** [comment]
I agree. It would be sensible to automatically be installed in the user's home directory or even better, provide a setting to specify the location.
